manifest.json

```diff
@@ -42,14 +46,15 @@
     },
     "content_scripts": [
       {
-        "matches": ["https://*.duckduckgo.com/*", "https://duckduckgo.com/*"],
+        "matches": ["<all_urls>"],
+        "all_frames": true,
         "css": ["css/noatb.css"]
       }
     ],
     "permissions": [
         "contextMenus",
         "webRequest",
         "webRequestBlocking",
-        "*://*.duckduckgo.com/"
+        "*://*/*"
```

Comment:
I'm not sure if it will, but unfortunately we need it to filter all urls and block trackers